docs/compute-systems/casper/compiling-code-on-casper/index.md

```diff
@@ -436,11 +436,10 @@ The sample below demonstrates how to compile CUDA C code on casper.
 
     **Log into a GPU-Enabled node**
 
-    Log in to either Casper or Derecho and run `execcasper` with a GPU resource request to start an interactive job on a GPU-accelerated Casper node:
+    Log in to either Casper or Derecho and run `qinteractive @casper` with a GPU resource request to start an interactive job on a GPU-accelerated Casper node:
 
     ```sh
-    execcasper -l gpu_type=gp100 --ngpus=1
-
+    qinteractive @casper -l gpu_type=gp100 --ngpus=1
```

Comment:
100% agreed. Here I just wanted to update the command to use `qinteractive` instead of `execcasper`.    Yes, some parts need revisiting when we have more permissive GPU spec options. Do you prefer any updates to this now? We can of course tell folks to compile on V100s instead but I think V100s are usually busier than gp100s?

Reply:
No let's defer. It looks good for now!